src/game/client/neo/ui/neo_hud_friendly_marker.h

```diff
@@ -1,15 +1,70 @@
-#ifndef NEO_HUD_FRIENDLY_MARKER_H
-#define NEO_HUD_FRIENDLY_MARKER_H
-#ifdef _WIN32
 #pragma once
-#endif
 
 #include "hudelement.h"
 #include <vgui_controls/Panel.h>
 
 #include "neo_gamerules.h"
 #include "neo_hud_worldpos_marker.h"
 
+enum NeoIFFMarkerSegment
+{
+	NEOIFFMARKER_SEGMENT_I_INITIALOFFSET = 0,
+	NEOIFFMARKER_SEGMENT_B_SHOWDISTANCE,
+	NEOIFFMARKER_SEGMENT_B_VERBOSEDISTANCE,
+	NEOIFFMARKER_SEGMENT_B_SHOWSQUADMARKER,
+	NEOIFFMARKER_SEGMENT_FL_SQUADMARKERSCALE,
+	NEOIFFMARKER_SEGMENT_B_SHOWHEALTHBAR,
+	NEOIFFMARKER_SEGMENT_B_SHOWHEALTH,
+	NEOIFFMARKER_SEGMENT_B_SHOWNAME,
+	NEOIFFMARKER_SEGMENT_B_PREPENDCLANTAGTONAME,
+	NEOIFFMARKER_SEGMENT_I_MAXNAMELENGTH,
+
+	NEOIFFMARKER_SEGMENT__TOTAL,
+};
+
+#define NEO_SQUAD_MARKER_DEFAULT "0;1;0;0;0.50;1;0;1;1;32;"
+#define NEO_FRIENDLY_MARKER_DEFAULT "0;1;0;1;0.50;1;0;0;0;32;"
+#define NEO_SPECTATOR_MARKER_DEFAULT "0;0;0;1;0.50;1;1;1;1;32;"
+#ifdef GLOWS_ENABLE
+#define NEO_SQUAD_XRAY_MARKER_DEFAULT "4;0;0;0;0.50;1;0;1;0;32;"
+#define NEO_FRIENDLY_XRAY_MARKER_DEFAULT "4;0;0;0;0.50;1;0;0;0;32;"
+#define NEO_SPECTATOR_XRAY_MARKER_DEFAULT "4;0;0;0;0.50;1;0;1;1;32;"
+#endif // GLOWS_ENABLE
+constexpr int NEO_IFFMARKER_SEQMAX = 32;
+constexpr int MAX_MARKER_STRLEN = 48 + NEO_MAX_CLANTAG_LENGTH + 1;
```

Comment:
Nit: could we rename this `..._SIZE` or something like that, because "strlen" typically means the length without the null terminator, so this could be confusing and lead to off-by-one memory bugs if someone incorrectly assumes it means `MAX_MARKER_STRLEN+1` is the null terminator.

E.g. `sizeof("foo") == 3+1`, vs `strlen("foo") == 3`